src/AbstractActionHandler.ts

```diff
@@ -33,6 +35,7 @@ export abstract class AbstractActionHandler {
     block: Block,
     blockMeta: BlockMeta,
     isReplay: boolean,
+    lastIrreversibleBlockNumber: number,
   ): Promise<number | null> {
```

Comment:
This function is starting to accumulate a lot of parameters, I wonder if there is a better way to do this

Reply:
Now that I think about it... we would be able to utilize the new interface talked about before here, combining `block`, `blockMeta`, and `lastIrreversibleBlock` into a single object.

Reply:
That would be nice, that would clean this function definition up a lot